data/extension/init/templates/tinygo/envoy.filters.network/default/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm"
+	"github.com/tetratelabs/proxy-wasm-go-sdk/proxywasm/types"
+)
+
+var (
+	connectionCounterName = "proxy_wasm_go.connection_counter"
+	counter               proxywasm.MetricCounter
+)
+
+type context struct{ proxywasm.DefaultContext }
```

Comment:
Let's add a comment explaining what `context` represents.

It is it an equivalent of a "Network Filter" in terms of Envoy ? What about lifecycle of `context` instance ?